src/components/about/About.jsx

```diff
@@ -1,9 +1,24 @@
 import React from 'react'
 import './About.css'
+import girl from "../../images/aboutImg.png"
 
 const About = () => {
     return (
-        <div>About</div>
+        <section id="about">
+            <div id="aboutLeft">
+                <h1 className="mainHeading">
+                    Обо мне
+                </h1>
+                <div id="aboutDesc">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
```

Comment:
For the future we can create separate file with constant and export them here.

E.g
constants.js will contain this 
`export const mockText = 'Lorem ipsum...'`
then in About.jsx 
`import { mockText } from 'constants.js'`
and insert in 'aboutDesc' section
```
<div id="aboutDesc">
    {mockText }
</div>
```

Thus, we will have a separate file with easily changeable text

Reply:
@vladBrownDev what do you think about it?